src/Yaml.ts

```diff
@@ -275,6 +290,6 @@ export function formatYaml(obj: any, options = { keepArrayIndent: false }): stri
  * @param indentArray retain indented arrays if `true`
  * @return YAML document as string with arrays indented as desired.
  */
-function arrayIndent(y: string, indentArray: boolean): string {
+function arrayIndent(y: string, indentArray: boolean | undefined): string {
```

Comment:
It surprises me that you have to have the undefined there. Can't anything be undefined?

Reply:
You have to because the properties in the interface are optionals and thus could be undefined.

The reason they are optionals is not to force them to be added on every function call and allow for overriding one only when needed.

Reply:
I know the value could be undefined, I'm just surprised the type system requires you to explicitly state that.